src/main/kotlin/me/partlysanestudios/partlysaneskies/features/gui/mainmenu/PSSMainMenu.kt

```diff
@@ -563,7 +563,7 @@ class PSSMainMenu : WindowScreen(ElementaVersion.V5) {
                 width = 100.percent
                 textScale = 1.33.scaledPixels
             }.setText(
-                "§8${announcement.date}§r\n§7${announcement.description}"
+                "§8${announcement.date}§r\n§7${announcement.description.replace("\n", "\n§7")}"
```

Comment:
Any chance we can get it to find the previous color? Because sometimes we add coloured main menu announcements

Reply:
We do that?

Reply:
I've done it at least twice. We support it lol.